broker/artemis/plugin/src/main/java/org/eclipse/kapua/broker/artemis/plugin/security/AddressMap.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.broker.artemis.plugin.security;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class AddressMap {
```

Comment:
To me "AddressMap" seems to imply that it is a "Map" implementation.  Maybe "AddressTracker" or "AddressAccessTracker" would be better?